.vscode/settings.json

```diff
@@ -12,11 +10,24 @@
   "editor.cursorWidth": 5,
   "editor.cursorBlinking": "solid",
   // Very important: Install this plugin: https://github.com/be5invis/vscode-custom-css
-  // you'll need to change this to an absolute path on your computer
-  "vscode_custom_css.imports": [
-    "/Volumes/Macintosh HD/Users/wesbos/.vscodestyles.css"
-  ],
-  "editor.renderWhitespace": "all",
+  // you'll need to change this to an absolute path on your computer,
+  "editor.renderWhitespace": "none",
   "workbench.editor.enablePreview": false,
-
-}
\ No newline at end of file
+  "window.zoomLevel": 1,
+  "workbench.iconTheme": "vscode-icons",
+  "terminal.integrated.shell.windows": "C:\\Program Files\\Git\\bin\\bash.exe",
+  "rust.rustup": {
+    "toolchain": "stable-x86_64-pc-windows-msvc",
+    "nightlyToolchain": "nightly-x86_64-pc-windows-msvc"
+  },
+  "[go]": {
+    "editor.insertSpaces": false,
+    "editor.formatOnSave": false
+  },
+  "rust.mode": "legacy",
+  "rust.actionOnSave": "check",
+  "vsicons.projectDetection.disableDetect": true,
+  "files.autoSave": "afterDelay",
+  "git.autofetch": true,
+  "workbench.colorTheme": "Cobalt2"
```

Comment:
why this is uploaded for the library

Reply:
When I change my laptop I can have all the vscode settings 😁😁